docs/Installation.md

```diff
@@ -8,26 +8,33 @@
 
 ## Configure PLC Connection
 
-To read data from the PLC and provide the data, we will use the app S7 Connector to establish a connection to the PLC via OPC UA. The S7 Connector publishes the data on the Databus, where the app Data Service can model and collect the energy data, that is needed. In order to build this infrastructure, these apps must be configured properly:
+To read data from the PLC and provide the data, we will configure the app OPC UA Connector with the Common Configurator to establish a connection to the PLC via OPC UA. The OPC UA Connector Connector publishes the data on the Databus, where the app Data Service can model and collect the energy data, that is needed. In order to build this infrastructure, these apps must be configured properly:
 
 - Databus
 - OPC UA Connector
+- Common Configurator
 - Data Service
 
-Please refer to [using the Data Service](https://github.com/industrial-edge/data-service) for detailed instructions.
+Please refer to [using the IIH](https://github.com/industrial-edge/data-service) for detailed instructions.
```

Comment:
Could you please adjust this link to: https://github.com/industrial-edge/iih-essentials-getting-started 
The repos is reachable via both URLs, but data-service is the legacy name,